Automattic/wp-calypso
# Stats: Gate currently open summary pages

Related to https://github.com/Automattic/red-team/issues/299

## Proposed Changes

Adds currently open Summary pages to the list of gated modules. The newly gated modules are:

- Posts & pages
- Countries
- File downloads

### Example of gated Summary page

<img width="733" alt="SCR-20241211-bkuz" src="https://github.com/user-attachments/assets/916a1d0e-0052-4759-accb-4e8f1981847b">

## Why are these changes being made?


The various summary pages are accessible via URL or indirectly via site switching. These changes make sure that upsells are properly displayed for the above listed modules.

## Testing Instructions



* Open the Calypso Live link.
* Visit the Traffic page for a new free site without a plan.
* Manually visit the three updated Summary pages and confirm the upsell is shown.

#### The partial paths to the Summary pages:
* `/stats/day/posts/SITE_SLUG`
* `/stats/day/countryviews/SITE_SLUG`
* `/stats/day/filedownloads/SITE_SLUG`

## Pre-merge Checklist
